recipes/cyan/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "cyan" %}
+{% set version = "0.8" %}
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: cyan  # [unix]
+  fn: cyan.exe  # [win]
+  # Linux
+  url: https://github.com/rwcarlsen/cyan/releases/download/v{{ version }}/cyan-linux-amd64 # [linux]
+  sha256: f343ae616c66cc7d27f42f9b68b25135b74188361dd0764de940ff7e2dd0bf2b  # [linux]
+  # Mac
+  url: https://github.com/rwcarlsen/cyan/releases/download/v{{ version }}/cyan-darwin-10.6-amd64  # [osx]
+  sha256: 5670d6ffa9d161eca66dee81115583998ef64a60a5534ebbcc2daaa04b9c81b4 # [osx]
+  # Windows
```

Comment:
If you remove the `# Linux`, and `# Windows` comments, that are getting interpreted as pre-selectors, the lint will work.
(I would remove the `# Mac` too b/c the pre-selector already brings that info and I would also remove the `fn:` b/c that is not needed.)

Reply:
Oh, hah! Weird. OK.

Reply:
Not weird, just buggy parsing, I figure that they were doing some `if line.lower().startswith(# win)`, so...